geps/gep-922.md

```diff
@@ -3,19 +3,24 @@
 * Issue: [#922](https://github.com/kubernetes-sigs/gateway-api/issues/922)
 * Status: Standard
 
+!!! note
```

Comment:
is `!!! note` intended to be as it is? I see no fancy markdown stuff being rendered.

Reply:
Yeah, it's rendered in the [deploy preview](https://deploy-preview-2446--kubernetes-sigs-gateway-api.netlify.app/geps/gep-922/) - this is a directive to `mkdocs`, not a standard Markdown one.